src/main/resources/jenkins/plugins/logstash/configuration/ElasticSearch/help-host.html

```diff
@@ -1,4 +1,4 @@
 <div>
   <p>The host name or IP address of the indexer to send log data to.<br>
-  ELASTICSEARCH: Also specify scheme. Example: "https://myserver". </p>
+  Also specify scheme. Example: "https://myserver". </p>
```

Comment:
can we add a separate scheme field instead?

Reply:
actually what would be even better is to have a single field accepting a full URL instead. Do you think it's possible?

Reply:
Then we could also remove the key, as this is just added to the url

Reply:
done